docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,411 @@
+
+##### Manual
+
+Copy the Virtual Host template as the new Apache configuration:
+
+```bash
+cp vendor/ibexa/post-install/resources/templates/apache2/vhost.template .ddev/apache/apache-site.conf
+```
+
+Then, replace the placeholders with the appropriate values in this `.ddev/apache/apache-site.conf`:
+
+| Placeholder         | Value                        |
+|---------------------|------------------------------|
+| `%IP_ADDRESS%`      | `*`                          |
+| `%PORT%`            | `80`                         |
+| `%HOST_NAME%`       | `my-ddev-project.ddev.site`  |
+| `%HOST_ALIAS%`      | `*.ddev.site`                |
+| `%BASEDIR%`         | `/var/www/html`              |
+| `%BODY_SIZE_LIMIT%` | `0`                          |
+| `%TIMEOUT%`         | `0`                          |
+| `%FASTCGI_PASS%`    | `unix:/var/run/php-fpm.sock` |
+
+You can, for example, do it with `sed`:
+
+```bash
+sed -i 's/%IP_ADDRESS%/*/' .ddev/apache/apache-site.conf
+sed -i 's/%PORT%/80/' .ddev/apache/apache-site.conf
+sed -i 's/%BASEDIR%/\/var\/www\/html/' .ddev/apache/apache-site.conf
+sed -i 's/%BODY_SIZE_LIMIT%/0/' .ddev/apache/apache-site.conf
+sed -i 's/%TIMEOUT%/0/' .ddev/apache/apache-site.conf
+sed -i 's/%FASTCGI_PASS%/unix:\/var\/run\/php-fpm.sock/' .ddev/apache/apache-site.conf
+```
+
+Finally, restart the project:
+
+```bash
+ddev restart
+```
+
+##### Scripted
+
+Generate the Virtual Host using https://github.com/ibexa/docker/blob/main/scripts/vhost.sh:
+
+```bash
+curl -O https://raw.githubusercontent.com/ibexa/docker/main/scripts/vhost.sh
+bash vhost.sh --template-file=vendor/ibexa/post-install/resources/templates/apache2/vhost.template \
+  --ip='*' \
+  --host-name='my-ddev-project.ddev.site' \
+  --host-alias='*.ddev.site' \
+  --basedir='/var/www/html' \
+  --sf-env=dev \
+  > .ddev/apache/apache-site.conf
+rm vhost.sh
+```
+
+Then, restart the project:
+
+```bash
+ddev restart
+```
+
+### Run an already existing project
+
+To run an existing project, you'll need to
+
+- configure the DDEV project
+- start the DDEV project
+- add Composer authentication if it's a Content, Experience or Commerce project
+- install dependencies packages using Composer
+- populate the contents, which could be
+  - getting a clean database using ddev `php bin/console ibexa:install ibexa-<edition>` then adding some data using [Ibexa data migration](../content_management/data_migration/importing_data.md)
+  - injecting a dump using [`ddev import-db`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#import-db) and copying related binary files into `public/var`
+
+This example will run an existing project and have the right content structure but no content. This following script sequence will
+
+- clone [version-controlled project](install_ibexa_dxp.md#add-project-to-version-control) from a Git repository,
```

Comment:
Is it possible to leave comments in bash? Then maybe moving each bullet point above the relevant line would make the example easier to understand and de-bullet point the text itself.

Reply:
Yes. Can be less readable but here the previous ordered list already introduce the strategy. Done in 2eab6f24c529bbf5e483c7bdaa23549d8535b1d1